### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by BenMukebo**
### Here are we have done to write unit tests for the Math Magicians app, using the Jest and React Testing Library libraries

- Set up React Testing Library.
- Follow the instructions in the official documentation.
- Write unit tests for the files operate.js and calculate.js using Jest.
- Create unit tests for all React components:
  - Use Jest snapshots to test the components.
  - Use React Testing Library to simulate user interaction.
- Run your full tests suite using Jest. All tests should pass.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @BenMukebo, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/__test__/Quote.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+// import isTSAnyKeyword from '@babel/types';
+// import { render } from '@testing-library/react';
```
Comment (mke2111) [inline src/components/__test__/Quote.test.js]: - [x] Remove these lines of commented-out code. It is always a good practice to keep your code clean from such comments.

Comment (BenMukebo) [reply src/components/__test__/Quote.test.js]: Thanks for the observation, the issue has been solved.

**Review by mke2111 — APPROVED** [review]
### Hi @BenMukebo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BenMukebo** [comment]
Thank you for the review.

**Pull request merged**